back/spec/models/initiative_spec.rb

```diff
@@ -266,13 +266,13 @@
       expect(User.find(cosponsor1.id)).to be_present
     end
 
-    it 'removes cosponsors_initiative even when an associated notifcation exists' do
-      cosponsor2 = create(:user)
-      create(:invitation_to_cosponsor_initiative, cosponsors_initiative: cosponsors_initiative)
-      initiative.update!(cosponsor_ids: [cosponsor2.id])
+    # it 'removes cosponsors_initiative even when an associated notifcation exists' do
```

Comment:
Can be deleted

Reply:
Deleted in part 3: https://github.com/CitizenLabDotCo/citizenlab/pull/9906